CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+
+# Branching for Contributors
+
+[OSCAL](https://github.com/usnistgov/OSCAL/) is the parent project of all `OSCAL-*` related repositories, and ***generally** all practices and guidance for contributions, versioning and branching apply.  
```

Comment:
suggestion:  "[OSCAL](https://github.com/usnistgov/OSCAL/) is the parent repository of all `OSCAL-*` related repositories, and, ***generally**, all practices and guidance for contributions, versioning and branching defined in OSCAL also apply to those repositories."